package/gluon-config-mode-mesh-vpn/luasrc/lib/gluon/config-mode/wizard/0300-mesh-vpn.lua

```diff
@@ -64,5 +62,11 @@ return function(form, uci)
 		os.execute('exec /lib/gluon/mesh-vpn/update-config')
 	end
 
-	return {'gluon', 'fastd', 'tunneldigger', 'simple-tc'}
+	local uci_sections = {'gluon', 'simple-tc'}
+
+	for _, section in ipairs(active_vpn.uci_sections()) do
+		table.insert(uci_sections, section)
+	end
+
+	return {'gluon', 'simple-tc'}
```

Comment:
Maybe we should first get rid of all these useless return values first and simply do a global `uci commit` at the end of the config wizard... It will only write files that need to change anyways.